server/auth/auth.go

```diff
@@ -30,6 +30,8 @@ func unixMsNow() time.Time {
 type Storage interface {
 	// CloseAccount closes the account for violation of the specified rule.
 	CloseAccount(account.AccountID, account.Rule) error
+	// OpenAccount opens an account that was closed by CloseAccount.
+	OpenAccount(account.AccountID) error
```

Comment:
Open is the opposite of close, so this makes sense, but "open account" feels like "create new account".  How about "restore", "reinstate", or even "reopen"?